thermalnetwork/tests/test_network.py

```diff
@@ -56,9 +56,7 @@ def test_network_two_ghe_area_proportional(self):
         )
 
         # -- Check
-        expected_depth_of_boreholes = 60
-        # FIXME: 60 is the minimum borehole length for a GHE (as set by sys-param file), which implies the loads
-        # were not correctly sent to GHED.
+        expected_depth_of_boreholes = pytest.approx(130, 1)
```

Comment:
We're getting actual values returned from GHED, so we must be giving it better information now. So we can test a real depth.